simmechanics/data/icub3/ICUB_3_upperbody_options.yaml

```diff
@@ -83,7 +83,56 @@ forceTorqueSensors:
           <plugin name="right_arm_ft_plugin" filename="libgazebo_yarp_forcetorque.so">
             <yarpConfigurationFile>model://iCub/conf/FT/gazebo_icub_right_arm_ft.ini</yarpConfigurationFile>
           </plugin>
-# TODO: fix different mapping of torso joints in controlboard
+assignedInertias:
+  # This is due to https://github.com/robotology/icub-models/issues/33
+  - linkName: r_shoulder_1
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: r_shoulder_2
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: r_shoulder_3
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: r_elbow_1
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: r_wrist_1
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: r_hand
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: l_shoulder_1
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: l_shoulder_2
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: l_shoulder_3
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: l_elbow_1
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: l_wrist_1
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01
+  - linkName: l_hand
+    xx: 0.01
+    yy: 0.01
+    zz: 0.01

```

Comment:
As this is the `ICUB_3_upperbody_options` file and not the `ICUB_3_upperbody_options_gazebo.yaml`, I guess this Gazebo-specific workaround can be removed. Otherwise it would be difficult to know which files  contain Gazebo specific workaround and which files do not.